.eslintrc.json

```diff
@@ -1,6 +1,21 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"],
+  "root": true,
+  "extends": [
+    "next/core-web-vitals",
+    "eslint:recommended",
+    "prettier",
+    "plugin:@typescript-eslint/recommended"
```

Comment:
Previously we also used this plugins:

'plugin:jest/recommended',
'plugin:react-hooks/recommended',
'airbnb',
'airbnb/hooks',
'airbnb-typescript'

Maybe there is a sense to add them as well. What do you think about that?

Reply:
Since there are existing `eslint:recommended` and `prettier` plugins I assume `airbnb` related plugins might be a little redundant (quite rules in common). What do you think?

Reply:
I agree